packages/cli/src/utils/platform.ts

```diff
@@ -0,0 +1,26 @@
+export const sanitizePath = (input: string): string => {
+  return input.replace(/[^a-zA-Z0-9 ._\-:\\/@]/g, '');
+};
+
+export const sanitizeLocale = (input: string): string => {
```

Comment:
Could you leave a comment with an explanation regarding what is being sanitized? Later, it could be hard to figure out the difference between the sanitizers 😄
Or perhaps you could come up with more descriptive names?